webapp/components/Editor/ContentViewer.vue

```diff
@@ -31,6 +31,9 @@ export default {
       }),
     }
   },
+  beforeUpdate() {
+    this.editor.setContent(this.content)
```

Comment:
Could you give me an idea what the effect of this is? :thinking:

Reply:
Yep! So, we're using the `tiptap editor` to display comments. The problem was that when the `content` prop changed the `ContentViewer` didn't update so the comment would always stay in its initial state (i.e. it could not be collapsed or expanded).

To solve this I had to a) use the `content` prop within the `<template>` to trigger an update at all – so I set it as a `key` and b) update the editor with the new `content` (which is happening above).

This was not necessary before because we were using three different instances of the `ContentViewer`.

Reply:
Thanks, sounds more like the VueJS way. :+1: